src/som/interpreter/actors/Actor.java

```diff
@@ -182,6 +275,11 @@ protected void logMessageAddedToMailbox(final EventualMessage msg) { }
   protected void logMessageBeingExecuted(final EventualMessage msg) { }
   protected void logNoTaskForActor() { }
   public long getActorId() { return 0; }
+  public long getReplayActorId() { return 0; }
```

Comment:
Ideally, all these methods are only on the `TracingActor`.
If we can move most or all code related to it into that class, we might not need any of this in the basic actor class.